Source/bmalloc/libpas/src/libpas/pas_malloc_stack_logging.h

```diff
@@ -64,7 +64,7 @@ static PAS_ALWAYS_INLINE bool pas_msl_is_enabled(void)
 
 static PAS_ALWAYS_INLINE pas_allocation_result pas_msl_malloc_logging(size_t size, pas_allocation_result result)
 {
-#if PAS_OS(DARWIN)
+#if PAS_OS(DARWIN) && !defined(__swift__) // FIXME: no malloc_logger for swift. see pas_darwin_spi.h
```

Comment:
To be consistent and avoid cluttering all our code with lots of explanation about what's still under development in Swift interop, I would just say "// Workaround for rdar://119319825".